app/controllers/cards_controller.rb

```diff
@@ -85,7 +86,7 @@ def destroy
   private
 
   def card_params
-    params.require(:card).permit(:name, :description, :due_date, assignee_ids: [])
+    params.require(:card).permit(:name, :description, :due_date, assignee_ids: []).merge(updated_at: Time.now)
```

Comment:
There's a comment about this in the PR.

I'd hear an argument to move it only into the `update` action though.

Reply:
I wanted to double check, it doesn't seem to change anything in regards to `#create` actions `created_at` and `updated_at` are the same on create.